apps/src/studio/bigGameInfo.js

```diff
@@ -0,0 +1,154 @@
+/* global Studio */
+
+var Direction = require('./constants').Direction;
+
+var BigGameInfo = function () {
+  this.functionNames = {};
+
+  this.playerSpriteIndex = 0;
+  this.targetSpriteIndex = 1;
+  this.dangerSpriteIndex = 2;
+};
+
+BigGameInfo.prototype.onTick = function () {
+   // Don't start until the title is over
+  var titleScreenTitle = document.getElementById('titleScreenTitle');
+  if (Studio.tickCount <= 1 ||
+      titleScreenTitle.getAttribute('visibility') === "visible") {
+    return;
+  }
+
+  // Update target, using onscreen and update_target
+  this.updateSpriteX_(this.targetSpriteIndex, this.update_target.bind(this));
+  // Update danger, using onscreen and update_danger
+  this.updateSpriteX_(this.dangerSpriteIndex, this.update_danger.bind(this));
+
+  // For every key and button down, call update_player
+  for (var key in Studio.keyState) {
```

Comment:
This key stuff might go away if we decide to use keydown? within update-player instead of sending it a key. Note, as the logic exists right now, we might call update-player multiple times per tick (if multiple keys are down).